### ISSUE TITLE
Button from ibm-tab-headers missing aria-label attribute

### ISSUE THREAD

**Issue by GSSCDylan** [carbon-design-system/carbon-components-angular#1989]
## Detailed description

> Describe in detail the issue you're having.

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug
> Is this issue related to a specific component?
tabs

> What browser are you working in?
Firefox 92.0.1 (64-bit)

> What version of the Carbon Design System are you using?
    "carbon-components": "10.31.0",
    "carbon-components-angular": "4.46.0",

## Steps to reproduce the issue

1. zoom in the panel that contains ibm-tab-header component 
2. Run Accessibility Assessment

## Additional information

* Screenshots or code
<img width="918" alt="button has no label" src="https://user-images.githubusercontent.com/73825731/153542041-ce197400-05f5-46dc-90f4-839b09440acf.png">
<img width="911" alt="aria - label" src="https://user-images.githubusercontent.com/73825731/153542051-34de4ba2-424a-4e00-9948-1d95db51f98c.png">

* Notes

**Comment by GSSCDylan** [comment]
Once a button has no aria-label, Jaws can not tell user the meaning of the button

**Comment by stanislavgeorgiev** [comment]
![image](https://github.com/carbon-design-system/carbon-components-angular/assets/1253469/d394380d-d379-4e00-b6e0-742f7ec67b83)

I see that the overflow button has a `title="Next"`. @GSSCDylan I think that you can close this issue
